docs/en/interfaces/third-party/integrations.md

```diff
@@ -76,7 +76,7 @@ ClickHouse, Inc. does **not** maintain the tools and libraries listed below and
         - [clickhouse-maxmind-geoip](https://github.com/AlexeyKupershtokh/clickhouse-maxmind-geoip)
 - AutoML
     - [MindsDB](https://mindsdb.com/)
-        - [MindsDB](https://github.com/mindsdb/mindsdb) - Predictive AI layer for ClickHouse database.
+        - [MindsDB](https://github.com/mindsdb/mindsdb) - The platform for customizing AI from enterprise data.
```

Comment:
How is it related to ClickHouse?

Reply:
Thank you @alexey-milovidov, I have updated it. Can you kindly have a look?